### PR TITLE
Added curl to send test stats to logstash.

### PR DESCRIPTION
**Pull request by jwfuller**
https://github.com/CuBoulder/w3_ansible/issues/35

### CODE REVIEW

File: tests/travis-ci/log-express-performance.php [hunk 1]
```diff
@@ -69,5 +90,51 @@
...
+
+    if (!$file) $status = -1;  // Site is down
+    else {
+        fclose($file);
+        $status = ($stoptime - $starttime) * 1000;
+        $status = floor($status);
+    }
+    print_r("fsock result: " . $status . "\n\n");
+  } else {
+    print_r("cURL result: " . $result . "\n\n");
```
Comment (alexfinnarn) [inline tests/travis-ci/log-express-performance.php]: I see "okcURL result: 1" on the Travis results page. I'm not sure where the "ok" part comes from unless cURL prints that out to the screen. Probably not a big deal.

File: tests/travis-ci/log-express-performance.php [hunk 2]
```diff
@@ -69,5 +90,51 @@
...
+  # If data is not getting into the logging stack, check the IP range of the
+  # TravisCI env. https://docs.travis-ci.com/user/ip-addresses/.
+  # Currently is
+  # workers-nat-org-shared-2.aws-us-east-1.travisci.net (52.45.185.117/32 52.54.31.11/32 54.87.185.35/32 54.87.141.246/32)
+  $ch = curl_init();
+  curl_setopt($ch, CURLOPT_URL, "http://wlogstash.colorado.edu:8080");
+  curl_setopt($ch, CURLOPT_CUSTOMREQUEST, "POST");
+  curl_setopt($ch, CURLOPT_POSTFIELDS, $data_string);
+  curl_setopt($ch, CURLOPT_TIMEOUT, 10);
+  curl_setopt($ch, CURLOPT_HTTPHEADER, array('Content-Type: application/json','User-Agent: TravisCI'));
```
Comment (alexfinnarn) [inline tests/travis-ci/log-express-performance.php]: Shouldn't there be some sort of authentication used for this POST request?

File: tests/travis-ci/log-express-performance.php [hunk 3]
```diff
@@ -69,5 +90,51 @@
...
+
+  if( ! $result = curl_exec($ch)) {
+    print_r("cURL error: " . curl_error($ch) . "\n\n");
+    $starttime = microtime(true);
+    $file      = fsockopen ('wlogstash.colorado.edu', 8080, $errno, $errstr, 10);
```
Comment (alexfinnarn) [inline tests/travis-ci/log-express-performance.php]: I assume $errno and $errstr are created from cURL receiving an error, but I don't see those variables declared in this file.

File: tests/travis-ci/log-express-performance.php (hunk 2 above)
Comment (jwfuller) [inline tests/travis-ci/log-express-performance.php]: Could add that, the port is IP restricted

File: tests/travis-ci/log-express-performance.php (hunk 3 above)
Comment (jwfuller) [inline tests/travis-ci/log-express-performance.php]: It will use defaults

File: tests/travis-ci/log-express-performance.php (hunk 1 above)
Comment (jwfuller) [inline tests/travis-ci/log-express-performance.php]: Yeah, not sure where that is coming from either

**Pull request merged**